decidim-core/spec/commands/decidim/update_account_spec.rb

```diff
@@ -77,6 +79,11 @@ module Decidim
         expect(user.reload.locale).to eq("es")
       end
 
+      it "updates the time zone" do
+        expect { command.call }.to broadcast(:ok)
+        expect(user.reload.time_zone).to eq("UTC")
+      end
+
```

Comment:
```suggestion
      it "updates the time zone" do
        expect { command.call }.to broadcast(:ok)
        expect(user.reload.time_zone).to eq("UTC")
      end

      context "when timezone is not defined" do
        let(:timezone) { "" }
        it "returns ok" do
           expect { command.call }.to broadcast(:ok)
           expect(user.reload.time_zone).to eq("")
        end
      end

      context "when timezone is invalid" do
        let(:timezone) { "giberish" }
        it "returns invalid" do
           expect { command.call }.to broadcast(:invalid)
        end
      end
```

We can add more test cases here, maybe the last one could be better tested in the `account_form_spec.rb`, as you preferer.